pkg/cluster/cluster.go

```diff
@@ -191,9 +197,15 @@ func (c *Cluster) Create(clusterSettings *ClusterSettings, clusterCache *cache.C
 
 	socatPath := path.Join(clusterCache.Dir(), "socat")
 	copyItems = append(copyItems, copyItem{dst: "/usr/bin/socat", src: socatPath})
-
-	copyItems = append(copyItems, copyItem{dst: "/opt/cni/bin/bridge", src: path.Join(clusterSettings.CNIPluginDir, "bridge")})
-	copyItems = append(copyItems, copyItem{dst: "/opt/cni/bin/loopback", src: path.Join(clusterSettings.CNIPluginDir, "loopback")})
+	files, err := ioutil.ReadDir(clusterSettings.CNIPluginDir)
+	if err != nil {
+		return errors.Errorf("no CNI plugins in %q", clusterSettings.CNIPluginDir)
+	}
+	for _, file := range files {
```

Comment:
I have set `cni-plugin-dir: /home/user/code/go/bin` in my kube-spawn config file, i.e. this code would copy all contents of `$GOPATH/bin`, no just the required CNI plugins.

Can you copy only the files that are actually needed?

Reply:
Um, I'm not sure how to do that and be generic, I can hardcode the current set of CNI plugins, but it may change.